pom.xml

```diff
@@ -117,8 +122,58 @@
                 <artifactId>json-schema-validator</artifactId>
                 <version>2.2.6</version>
             </dependency>
-        </dependencies>
+            <dependency>
```

Comment:
I don't see any new references to any of these dependencies.  What needed them?  Are they really needed in dependency management?

Reply:
It was required, otherwise the docker build was failing. Now it is not required since we have moved the contents to ```connectors``` ```pom.xml```.